inference-engine/include/ie_core.hpp

```diff
@@ -173,9 +173,18 @@ class INFERENCE_ENGINE_API_CLASS(Core) {
      * operation*
      * @return An executable network reference
      */
-    ExecutableNetwork ImportNetwork(std::istream& networkModel, const std::string& deviceName = {},
+    ExecutableNetwork ImportNetwork(std::istream& networkModel, const std::string& deviceName,
```

Comment:
Maybe we should also add test for 'empty device name' + handle 'if (!deviceName.empty()' in implementation

Reply:
added `smoke_ImportNetworkThrowsIfNoDeviceName` test